docs/self-managed/operational-guides/configure-flow-control/configure-flow-control.md

```diff
@@ -0,0 +1,311 @@
+---
+id: configure-flow-control
+title: "Configure Flow Control"
+sidebar_label: "Configure Flow Control"
+description: "Learn how to configure Flow Control"
+---
+
+# Configuring Flow Control
+
+When a broker receives a client request, it is written to the **_event
+stream_** first (see section [internal processing](/components/zeebe/technical-concepts/internal-processing.md) for details), and processed later by the stream processor.
+
+If the exporting is slow, if there are many client requests in the stream,
+or processing largely outpaces the exporting rate, it can lead to the backlog
+increasing and the processing latency can grow beyond an acceptable time.
```

Comment:
:wrench: This mixes two different things: processing backlog and exporting backlog. Please separate them to make this a bit less confusing.

Reply:
Here I seperated both points in two distinct sentences.